scaffolding-node/test/controls/scaffolding_node_spec.rb

```diff
@@ -1,198 +0,0 @@
-# encoding: utf-8
```

Comment:
I've preserved these tests at https://github.com/smith/habitat-node-scaffolding-inspec-profile in case we want to be able to verify the behavior of this scaffolding in the future.